### PR TITLE
GSdx: Add an enum for channel fetch

### PR DESCRIPTION
**Pull request by lightningterror**
Maybe both ShaderConvert and ChannelFetch enums can be moved to GS.h.

@tadanokojin I believe you were looking for something like this ?

### CODE REVIEW

**Comment by tadanokojin** [comment]
That's what I was looking for thank you.

File: plugins/GSdx/GSDevice.h [hunk 1]
```diff
@@ -52,6 +53,17 @@ enum ShaderConvert {
 	ShaderConvert_Count
 };
 
+enum ChannelFetch
+{
+	ChannelFetch_NONE  = 0,
```
Comment (willkuer) [inline plugins/GSdx/GSDevice.h]: Thats camel case, snake case and uppercase all in one line. Enum classes are imo a better way to enable namespacing on enums than prefixes.

**Comment by ssakash** [comment]
I don't know why you didn't change it into an enum class as willkuer said.

**Comment by ssakash** [comment]
1/ You don't need the pseduo namespace when using enum class, that's the exact point on using it.
2/ I think there's a generic enum cast function available instead of doing ``static_cast<int>()``

**Pull request merged**